test/support/pages/SettingsPage.js

```diff
@@ -0,0 +1,331 @@
+// in test/support/pages/SettingsPage.js
+define(function (require) {
+  // the page object is created as a constructor
+  // so we can provide the remote Command object
+  // at runtime
+
+  var Promise = require('bluebird');
+  var Common = require('./Common');
+
+  var defaultTimeout = 5000;
+  var common;
+
+  function SettingsPage(remote) {
+    this.remote = remote;
+    common = new Common(this.remote);
+  }
+
+  SettingsPage.prototype = {
+    constructor: SettingsPage,
+
+    navigateTo: function () {
+      return common.navigateToApp('settings');
+    },
+
+    getTimeBasedEventsCheckbox: function getTimeBasedEventsCheckbox() {
```

Comment:
I don't think so. @Bargs mentioned that as well. @LeeDr originally did that to try to get cleaner stack traces, but the consensus seems to be that it's more confusing than anything else - I'll clean them up.